diff --git a/chrome/browser/history_embeddings/history_embeddings_utils.cc b/chrome/browser/history_embeddings/history_embeddings_utils.cc
index 8198ba0..02e39df 100644
--- a/chrome/browser/history_embeddings/history_embeddings_utils.cc
+++ b/chrome/browser/history_embeddings/history_embeddings_utils.cc
@@ -49,15 +49,8 @@ bool IsCountryAndLocale(const std::string& country, const std::string& locale) {
 
 }  // namespace
 
-constexpr auto kEnabledByDefaultForDesktopOnly =
-#if BUILDFLAG(IS_ANDROID)
-    base::FEATURE_DISABLED_BY_DEFAULT;
-#else
-    base::FEATURE_ENABLED_BY_DEFAULT;
-#endif
-
 // These are the kill switches for the launched history embeddings features.
-BASE_FEATURE(kLaunchedHistoryEmbeddings, kEnabledByDefaultForDesktopOnly);
+BASE_FEATURE(kLaunchedHistoryEmbeddings, base::FEATURE_DISABLED_BY_DEFAULT);
 
 bool IsHistoryEmbeddingsEnabledForProfile(Profile* profile) {
   if (!IsHistoryEmbeddingsFeatureEnabled()) {
diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index e9f21e3..dab03af 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -584,14 +584,7 @@ BASE_FEATURE(kGlicMessageFirstFre, base::FEATURE_DISABLED_BY_DEFAULT);
 // IMPORTANT: this feature should never be expired! It is used as the main
 // kill-switch for Glic and can be used in the future to handle unsupported
 // Chrome versions.
-BASE_FEATURE(kGlic,
-#if BUILDFLAG(IS_MAC) || BUILDFLAG(IS_WIN) || BUILDFLAG(IS_CHROMEOS) || \
-    BUILDFLAG(IS_ANDROID)
-             base::FEATURE_ENABLED_BY_DEFAULT
-#else
-             base::FEATURE_DISABLED_BY_DEFAULT
-#endif
-);
+BASE_FEATURE(kGlic, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kGlicBackgroundActuation, base::FEATURE_DISABLED_BY_DEFAULT);
 
diff --git a/components/compose/core/browser/compose_features.cc b/components/compose/core/browser/compose_features.cc
index bc0d7b5..2c8be91 100644
--- a/components/compose/core/browser/compose_features.cc
+++ b/components/compose/core/browser/compose_features.cc
@@ -20,7 +20,7 @@ BASE_FEATURE(kComposeAXSnapshot, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kComposeAutoSubmit, base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kComposeEligible, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kComposeEligible, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kEnableComposeSavedStateNudge,
              "ComposeNudge",
diff --git a/components/lens/lens_features.cc b/components/lens/lens_features.cc
index 5775195..e35f7f8 100644
--- a/components/lens/lens_features.cc
+++ b/components/lens/lens_features.cc
@@ -20,13 +20,7 @@ BASE_FEATURE(kLensStandalone, base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kShowContextualTasksMenuIcon, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kLensOverlay,
-#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_IOS)
-             base::FEATURE_DISABLED_BY_DEFAULT
-#else
-             base::FEATURE_ENABLED_BY_DEFAULT
-#endif
-);
+BASE_FEATURE(kLensOverlay, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kLensOverlayAndroid, base::FEATURE_DISABLED_BY_DEFAULT);
 
diff --git a/components/omnibox/browser/aim_eligibility_service_features.cc b/components/omnibox/browser/aim_eligibility_service_features.cc
index c23cef8..254308e 100644
--- a/components/omnibox/browser/aim_eligibility_service_features.cc
+++ b/components/omnibox/browser/aim_eligibility_service_features.cc
@@ -8,7 +8,7 @@
 
 namespace omnibox {
 
-BASE_FEATURE(kAimEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kAimEnabled, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimServerEligibilityEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
 
