diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -302,7 +302,7 @@
         </else>
       </if>
       <message name="IDS_APP_MENU_YOUR_CHROME_HEADER" desc="Header title for the Your Chromium section in the main App Menu">
-        Your Chromium
+        Your OpenArc
       </message>
       <if expr="is_win">
         <message name="IDS_SXS_SHORTCUT_NAME" desc="Unused in Chromium builds" translateable="false">
@@ -320,7 +320,7 @@
       </if>
       <if expr="is_macosx or is_linux">
         <message name="IDS_FIRST_RUN_DIALOG_WINDOW_TITLE" desc="Window title of First Run dialog on Mac and Linux, displayed in title bar">
-          Welcome to Chromium
+          Welcome to OpenArc
         </message>
       </if>
       <if expr="is_linux">
@@ -452,7 +452,7 @@
           </then>
           <else>
             <message name="IDS_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for titles displayed in tabs and popup windows">
-              <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - Chromium
+              <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - OpenArc
             </message>
           </else>
         </if>
@@ -491,7 +491,7 @@
         </then>
         <else>
           <message name="IDS_ACCESSIBLE_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for the accessible name of a tabbed browser window">
-            <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - Chromium
+            <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - OpenArc
           </message>
         </else>
       </if>
@@ -576,7 +576,7 @@
         </message>
       </if>
       <message name="IDS_ACCNAME_APP" desc="The accessible name for the app menu." translateable="false">
-        Chromium
+        OpenArc
       </message>
       <!-- Hung Browser Detector -->
       <if expr="is_win">
@@ -1019,7 +1019,7 @@
       <if expr="enable_extensions_core">
         <!-- Extension installed bubble -->
         <message name="IDS_EXTENSION_INSTALLED_HEADING" desc="Title of the extension-installed bubble. Instructs that the extension was installed.">
-          <ph name="EXTENSION_NAME">$1<ex>Gmail Checker</ex></ph> has been added to Chromium
+          <ph name="EXTENSION_NAME">$1<ex>Gmail Checker</ex></ph> has been added to OpenArc
         </message>
         <if expr="is_android">
           <message name="IDS_EXTENSION_INSTALLED_MANAGE_INFO_IN_MAIN_MENU" desc="Text displayed in the post install dialog when the extensions menu is unpinned on Android informing them they can manage their extensions in the main menu.">
@@ -1029,7 +1029,7 @@
 
         <!-- Extension uninstall prompt -->
         <message name="IDS_EXTENSION_UNINSTALL_PROMPT_REMOVE_DATA_CHECKBOX" desc="Checkbox text to ask the user whether they want to remove associated data at uninstall time. Only used when uninstalling an app associated with a particular website.">
-          Also delete data from Chromium (<ph name="URL">$1<ex>www.google.com</ex></ph>)
+          Also delete data from OpenArc (<ph name="URL">$1<ex>www.google.com</ex></ph>)
         </message>
 
         <!-- Extension alerts. -->
@@ -1052,7 +1052,7 @@
           Remove <ph name="EXTENSION_NAME">$1<ex>Google Translate</ex></ph> from Chromium
         </message>
         <message name="IDS_EXTENSIONS_MENU_ZERO_STATE_DESC" desc="Description in the Extensions Menu when there are no extensions installed." formatter_data="android_java">
-          Extensions are add-ons that let you customize Chromium. From staying organized, to simplifying everyday tasks, extensions help you make Chromium your own.
+          Extensions are add-ons that let you customize OpenArc. From staying organized, to simplifying everyday tasks, extensions help you make OpenArc your own.
         </message>
         <message name="IDS_OPEN_WEB_STORE_BUTTON_TEXT" desc="Text for the open Chrome Web Store button." formatter_data="android_java">
           Open Chrome Web Store
@@ -1092,19 +1092,19 @@
      other {Chromium recommends that you remove them}}
   </message>
       <message name="IDS_EXTENSIONS_ITEM_SHOW_ACCESS_REQUESTS_IN_TOOLBAR" desc="The label on the toggle to allow the extension to show site access requests in the toolbar.">
-        Allow extension to show access requests in the Chromium toolbar
+        Allow extension to show access requests in the OpenArc toolbar
       </message>
       <message name="IDS_EXTENSIONS_INCOGNITO_WARNING" desc="Warns the user that Chromium cannot prevent extensions from recording history in Incognito mode. Displayed in extensions management UI after an extension is selected to be run in Incognito mode.">
         Warning: Chromium cannot prevent extensions from recording your browsing history. To disable this extension in Incognito mode, unselect this option.
       </message>
       <message name="IDS_EXTENSIONS_UNINSTALL" desc="The label for a control that triggers an extension uninstall.">
-        Remove from Chromium
+        Remove from OpenArc
       </message>
       <message name="IDS_EXTENSIONS_SHORTCUT_SCOPE_IN_CHROME" desc="The label to indicate that a shortcut will be triggerable only from within the Chrome application.">
-        In Chromium
+        In OpenArc
       </message>
       <message name="IDS_PENDING_CHANGE_WARNING" desc="The message shown to users when an extension setting change will ba applied after Chrome restarts.">
-        Changes to this setting will be applied once Chromium restarts.
+        Changes to this setting will be applied once OpenArc restarts.
       </message>
 
       <if expr="enable_extensions">
@@ -1123,15 +1123,15 @@
 
       <!-- Main Chrome menu -->
       <message name="IDS_APPMENU_TOOLTIP" desc="The tooltip to show for the browser menu">
-        Customize and control Chromium
+        Customize and control OpenArc
       </message>
 
       <message name="IDS_APPMENU_TOOLTIP_UPDATE_AVAILABLE" desc="The tooltip to show for the browser menu when an update is available">
-        Customize and control Chromium. Update is available.
+        Customize and control OpenArc. Update is available.
       </message>
 
       <message name="IDS_APPMENU_TOOLTIP_ALERT" desc="The tooltip to show for the browser menu when a non-update alert is displayed">
-        Customize and control Chromium. Something needs your attention - click for details.
+        Customize and control OpenArc. Something needs your attention - click for details.
       </message>
 
       <message name="IDS_OPEN_IN_CHROME" desc="The text label of the Open in Chrome menu item for the Hosted App app menu">
@@ -1147,7 +1147,7 @@
           </then>
           <else>
             <message name="IDS_ABOUT" desc="In Title Case: The text label of the About Chrome menu item">
-              About &amp;Chromium
+              About &amp;OpenArc
             </message>
           </else>
         </if>
@@ -1164,7 +1164,7 @@
           </then>
           <else>
             <message name="IDS_ABOUT" desc="The text label of the About Chrome menu item">
-              About &amp;Chromium
+              About &amp;OpenArc
             </message>
           </else>
         </if>
@@ -1196,13 +1196,13 @@
       <!-- Page menu. Android has separate strings for its menu -->
       <if expr="not is_android">
         <message name="IDS_CUSTOMIZE_CHROME" desc="The text label of the Customize Chrome menu item">
-          C&amp;ustomize your Chromium
+          C&amp;ustomize your OpenArc
         </message>
         <message name="IDS_OTHER_CHROME_PROFILES_TITLE" desc="The title of the Other Chrome profiles section">
-          Other Chromium profiles
+          Other OpenArc profiles
         </message>
         <message name="IDS_MANAGE_CHROME_PROFILES" desc="The text label of the manage chrome profile item">
-          &amp;Manage Chromium profiles
+          &amp;Manage OpenArc profiles
         </message>
       </if>
 
@@ -1232,13 +1232,13 @@
 
       <!-- Session Restore Infobar -->
       <message name="IDS_SESSION_RESTORE_TURN_OFF_RESTORE_FROM_SESSION" desc="Message that appears in the session restore infobar when the user has restored their tabs once.">
-        Continue where you left off: Chromium restored your tabs from your last session. You can turn this off in
+        Continue where you left off: OpenArc restored your tabs from your last session. You can turn this off in
       </message>
       <message name="IDS_SESSION_RESTORE_TURN_OFF_RESTORE_FROM_RESTART" desc="Message that appears in the session restore infobar when the user has restored their tabs multiple times.">
-        Continue where you left off: Chromium restores your tabs every time you restart. To turn this off, go to
+        Continue where you left off: OpenArc restores your tabs every time you restart. To turn this off, go to
       </message>
       <message name="IDS_SESSION_RESTORE_TURN_ON" desc="Message that appears in the session restore infobar to get the user to opt-in into session restore.">
-        Continue where you left off: Chromium can restore your tabs every time you restart. To turn this on, go to
+        Continue where you left off: OpenArc can restore your tabs every time you restart. To turn this on, go to
       </message>
       <message name="IDS_SESSION_RESTORE_LINK" desc="Text for the link in the session restore infobar to open the Settings page.">
          Settings.
@@ -1318,7 +1318,7 @@
 
       <!-- NTP strings -->
       <message name="IDS_NTP_CUSTOMIZE_TOOLBAR_CATEGORY_YOUR_CHROME" desc="The label for the category of actions that involve your Chrome data, such as opening the password manager or clearing browsing data.">
-        Your Chromium
+        Your OpenArc
       </message>
       <if expr="_is_chrome_for_testing_branded">
         <then>
@@ -1328,7 +1328,7 @@
         </then>
         <else>
           <message name="IDS_NTP_CUSTOMIZE_BUTTON_LABEL" desc="Label for button on the New Tab Page that opens dialog to customize Chromium browser.">
-            Customize Chromium
+            Customize OpenArc
           </message>
         </else>
       </if>
@@ -1336,13 +1336,13 @@
         Make Chromium Yours
       </message>
       <message name="IDS_NTP_CUSTOMIZE_CHROME_RESET_TO_CLASSIC_CHROME_LABEL" desc="The label for the button for resetting theme back to default chrome in the New Tab Page customize chrome side panel.">
-        Reset to Default Chromium
+        Reset to Default OpenArc
       </message>
 
       <!-- New Tab Page browser feature promotions. -->
       <if expr="not is_android">
         <message name="IDS_NTP_CUSTOMIZATION_PROMO" desc="The text shown in the body of the New Tab Page browser customization promotion">
-          Customize Chromium to give your browser a new look
+          Customize OpenArc to give your browser a new look
         </message>
         <message name="IDS_NTP_MOBILE_PROMO_TITLE" desc="Title of the NTP promo that promotes the browser on mobile.">
           Get Chromium on your phone
@@ -1354,10 +1354,10 @@
 
       <!-- Customize Chrome Auto Promo Strings -->
       <message name="IDS_IPH_CUSTOMIZE_CHROME_AUTO_OPEN_BODY" desc="The description of the IPH to auto open Customize Chromium">
-        Customize Chromium to give it a new look
+        Customize OpenArc to give it a new look
       </message>
       <message name="IDS_IPH_CUSTOMIZE_CHROME_AUTO_OPEN_SCREENREADER" desc="The screenreader of the IPH to auto open Customize Chromium" is_accessibility_with_no_ui="true">
-        Click the Customize Chromium button on the New Tab Page to give Chromium a new look
+        Click the Customize OpenArc button on the New Tab Page to give OpenArc a new look
       </message>
 
       <!-- Realbox Strings -->
@@ -1379,7 +1379,7 @@
       <if expr="not use_titlecase">
         <message name="IDS_AVATAR_BUTTON_INTERCEPT_BUBBLE_CHROME_SIGNIN_TEXT"
           desc="Text asking the user to sign into Chromium while the Chromium Signin Bubble intercept is shown.">
-          Sign in to Chromium?
+          Sign in to OpenArc?
         </message>
         <message name="IDS_SIGNIN_DICE_WEB_INTERCEPT_BUBBLE_CHROME_SIGNIN_TITLE" desc="Title of the Chromium Signin Bubble intercept that is shown when a signed out user signs in on the web with it's first account.">
           Make Chromium your own
@@ -1392,7 +1392,7 @@
       <if expr="use_titlecase">
         <message name="IDS_AVATAR_BUTTON_INTERCEPT_BUBBLE_CHROME_SIGNIN_TEXT"
           desc="Text asking the user to sign into Chromium while the Chromium Signin Bubble intercept is shown.">
-          Sign In to Chromium?
+          Sign In to OpenArc?
         </message>
         <message name="IDS_SIGNIN_DICE_WEB_INTERCEPT_BUBBLE_CHROME_SIGNIN_TITLE" desc="Title of the Chromium Signin Bubble intercept that is shown when a signed out user signs in on the web with it's first account.">
           Make Chromium Your Own
@@ -1515,10 +1515,10 @@
       <if expr="not is_chromeos and not is_android">
         <!-- Profile Customization Web UI -->
         <message name="IDS_PROFILE_CUSTOMIZATION_TEXT" desc="Text of the profile customization bubble">
-          Customize your new Chromium profile
+          Customize your new OpenArc profile
         </message>
         <message name="IDS_PROFILE_CUSTOMIZATION_TITLE_V2" desc="Title of the profile customization dialog">
-          Customize your Chromium profile
+          Customize your OpenArc profile
         </message>
       </if>
 
@@ -1555,45 +1555,45 @@
 
       <!-- Profile Menu -->
       <message name="IDS_PROFILE_MENU_SIGNIN_PROMO_BUTTON" meaning="Button" desc="Text for the button in the sign-in promo in the profile menu when the used is signed out.">
-        Sign in to Chromium
+        Sign in to OpenArc
       </message>
       <message name="IDS_PROFILE_MENU_PLACEHOLDER_PROFILE_NAME" desc="Placeholder profile name, used when the user did not enter a name for their profile.">
-        Your Chromium
+        Your OpenArc
       </message>
       <if expr="not use_titlecase">
         <message name="IDS_PROFILE_MENU_SIGN_OUT_WHEN_SIGNIN_PENDING" desc="The text label of the Sign out of Chromium button in the Profile Menu when the user is in sign in pending state.">
-          Remove account from Chromium
+          Remove account from OpenArc
         </message>
         <message name="IDS_PROFILE_MENU_PROFILES_LIST_TITLE" desc="Title in the profile menu for the list of 'other Chromium profiles'.">
-          Other Chromium profiles
+          Other OpenArc profiles
         </message>
         <message name="IDS_PROFILE_MENU_MANAGE_PROFILES" desc="The text label of the Manage Chromium Profiles item in the Profile Menu.">
-          Manage Chromium profiles
+          Manage OpenArc profiles
         </message>
         <message name="IDS_PROFILE_MENU_SIGN_OUT"
           desc="The text label of the Sign out of Chromium button in the Profile Menu.">
-          Sign out of Chromium
+          Sign out of OpenArc
         </message>
         <message name="IDS_PROFILE_MENU_ADD_PROFILE" desc="The text label of the add new profile item in the profile menu.">
-          Add Chromium profile
+          Add OpenArc profile
         </message>
       </if>
       <if expr="use_titlecase">
         <message name="IDS_PROFILE_MENU_SIGN_OUT_WHEN_SIGNIN_PENDING" desc="The text label of the Sign out of Chromium button in the Profile Menu when the user is in sign in pending state.">
-          Remove Account from Chromium
+          Remove Account from OpenArc
         </message>
         <message name="IDS_PROFILE_MENU_PROFILES_LIST_TITLE" desc="Title in the profile menu for the list of 'other Chromium profiles'.">
-          Other Chromium Profiles
+          Other OpenArc Profiles
         </message>
         <message name="IDS_PROFILE_MENU_MANAGE_PROFILES" desc="The text label of the Manage Chromium Profiles item in the Profile Menu.">
-          Manage Chromium Profiles
+          Manage OpenArc Profiles
         </message>
         <message name="IDS_PROFILE_MENU_SIGN_OUT"
           desc="The text label of the Sign out of Chromium button in the Profile Menu.">
-          Sign Out of Chromium
+          Sign Out of OpenArc
         </message>
         <message name="IDS_PROFILE_MENU_ADD_PROFILE" desc="The text label of the add new profile item in the profile menu.">
-          Add Chromium Profile
+          Add OpenArc Profile
         </message>
       </if>
 
@@ -1669,7 +1669,7 @@
       <!-- Mac First-run dialog messages -->
       <if expr="is_macosx">
         <message name="IDS_FIRSTRUN_DLG_MAC_SET_DEFAULT_BROWSER_LABEL" desc="Label for checkbox that sets the default browser">
-          Set Chromium as your default browser
+          Set OpenArc as your default browser
         </message>
       </if>
 
@@ -1807,12 +1807,12 @@
         <!-- Sign-in Promo Avatar Button Label Desktop -->
         <if expr="not use_titlecase">
           <message name="IDS_AVATAR_BUTTON_SIGNIN_PROMO" desc="The avatar button promo label inviting the user to sign-in. Shown for signed out users, and signed in to the web only. When clicked, it opens the profile menu.">
-            Sign in to Chromium?
+            Sign in to OpenArc?
           </message>
         </if>
         <if expr="use_titlecase">
            <message name="IDS_AVATAR_BUTTON_SIGNIN_PROMO" desc="The avatar button promo label inviting the user to sign-in. Shown for signed out users, and signed in to the web only. When clicked, it opens the profile menu.">
-            Sign In to Chromium?
+            Sign In to OpenArc?
           </message>
         </if>
       </if>
@@ -2001,7 +2001,7 @@
 
       <!-- Side panel strings -->
       <message name="IDS_SIDE_PANEL_CUSTOMIZE_CHROME_TITLE" desc="The name of the Customize Chrome feature in the side panel combo box.">
-        Customize Chromium
+        Customize OpenArc
       </message>
 
       <!-- History Strings -->
@@ -2617,16 +2617,16 @@
         </message>
         <!-- Profile Picker -->
         <message name="IDS_PROFILE_PICKER_MAIN_VIEW_TITLE" desc="Profile picker main view title">
-          Welcome to Chromium profiles
+          Welcome to OpenArc profiles
         </message>
         <message name="IDS_PROFILE_PICKER_MAIN_VIEW_TITLE_V2" desc="Title of a screen that lets the user choose which profile to use when browsing the web. *If the EN source doesn't translate well in your language, an alternative string 'Which Chromium profile would you like to use?' may be used instead">
-          Who's using Chromium?
+          Who's using OpenArc?
         </message>
         <message name="IDS_PROFILE_PICKER_MAIN_VIEW_SUBTITLE" desc="Profile picker main view subtitle">
-          With Chromium profiles, you can separate all your Chromium stuff. Create profiles for friends and family, or split between work and fun.
+          With OpenArc profiles, you can separate all your OpenArc stuff. Create profiles for friends and family, or split between work and fun.
         </message>
         <message name="IDS_PROFILE_PICKER_PROFILE_CREATION_FLOW_PROFILE_TYPE_CHOICE_TITLE" desc="Profile picker profile type choice title">
-          Set up your new Chromium profile
+          Set up your new OpenArc profile
         </message>
         <message name="IDS_PROFILE_PICKER_IPH_FOR_PROFILES_TEXT" desc="Text of the IPH bubble for introducing profiles.">
           Each profile holds its own Chromium info like bookmarks, history, passwords, and more
@@ -2646,26 +2646,26 @@
 
         <!-- First Run Experience -->
         <message name="IDS_FRE_WELCOME_START_BUTTON_LABEL" desc="This string is found on a primary button that takes the user into the main browser experience. It is a verb-based call to action. (CHAR LIMIT = 20)">
-          Start Chromium
+          Start OpenArc
         </message>
 
         <message name="IDS_FRE_SIGN_IN_TITLE_0" meaning="Title"
           desc="This string appears as a heading on a full-page screen that asks the user to sign in to Chromium with their Google Account. It appears when the user launches Chromium for the first time. When they sign in, they can personalize and customize Chromium and remember their settings in their Google Account. The tone should be inviting and alluring.">
-          Sign in to Chromium
+          Sign in to OpenArc
         </message>
         <message name="IDS_FRE_GET_YOUR_BROWSER_READY_TITLE" meaning="Title" desc="It appears when the user launches Chromium for the first time on the page that is displayed before the sign in page. When they sign in, they can personalize and customize Chromium and remember their settings in their Google Account. The tone should be inviting and alluring.">
-          Let's get your Chromium ready
+          Let's get your OpenArc ready
         </message>
 
         <!-- Default Browser Promo -->
         <message name="IDS_FRE_DEFAULT_BROWSER_TITLE" desc="Title for the page that prompts user to set Chromium as their default browser in the first run experience.">
-          Set Chromium as your default browser
+          Set OpenArc as your default browser
         </message>
         <message name="IDS_FRE_DEFAULT_BROWSER_AND_PINNING_TITLE" desc="Title for the page that prompts user to set Chromium as their default browser and pin to taskbar in the first run experience.">
           Use Chromium by default and get it pinned
         </message>
         <message name="IDS_FRE_DEFAULT_BROWSER_SUBTITLE_NEW" desc="Subtitle for the page that prompts user to set Chromium as their default browser in the first run experience.">
-          Use Chromium anytime you click links in messages, documents, and other apps
+          Use OpenArc anytime you click links in messages, documents, and other apps
         </message>
         <message name="IDS_FRE_DEFAULT_BROWSER_AND_PINNING_SUBTITLE" desc="Subtitle for the page that prompts user to set Chromium as their default browser and pin to taskbar in the first run experience.">
           Open links in Chromium from any app. Plus for easy access, it gets pinned to your taskbar.
@@ -3325,12 +3325,12 @@
       <if expr="not is_android">
         <if expr="use_titlecase">
           <message name="IDS_PROFILE_MENU_ACCOUNT_SETTINGS_BUTTON" desc="Button in the avatar menu bubble view for opening Chrome account settings.">
-            Your Account in Chromium
+            Your Account in OpenArc
           </message>
         </if>
         <if expr="not use_titlecase">
           <message name="IDS_PROFILE_MENU_ACCOUNT_SETTINGS_BUTTON" desc="Button in the avatar menu bubble view for opening Chrome account settings.">
-            Your account in Chromium
+            Your account in OpenArc
           </message>
         </if>
       </if>
